tsconfig.json

```diff
@@ -44,5 +44,5 @@
     "utils/test/jest.d.ts"
   ],
   "include": ["packages/*/src/**/*", "packages/*/demo/**/*"],
-  "exclude": ["node_modules", "**/node_modules", "**/dist", "packages/.template/**/*"]
+  "exclude": ["node_modules", "**/node_modules", "**/dist"]
```

Comment:
Odd, this doesn't seem to be reflected in the "files changed".. for my understanding, what's the reason for removing `"packages/.template/**/*"`? Was it causing issues?

Reply:
I added and removed this change in the current PR. Turns out it doesn't address the TypeScript warnings in CI. I'll leave addressing the warnings to a separate task since the warnings have been around before this PR.